polyply/tests/test_data/library_tests/2016H66/PVA/polyply/PVA.itp

```diff
@@ -132,4 +134,5 @@ PVA 3
  7  8 10  6 2 gi_2 ; center S
 11 12 14 10 2 gi_5 ; center R
 15 16 18 14 2 gi_5 ; center R
-19 18 20 22 2 gi_2 ; center S
+19 20 22 18 2 gi_2 ; center S
```

Comment:
this is a different dihedral - so just double-checking that this change was intended, right?

Reply:
This one is a good catch, because it got me thinking a bit about the library check. To quickly answer the question, the change is intended. It aligns the reference file with what polyply produces.

Effectively these dihedrals are actually the same. That is because this dihedral describes a tetrahedral configuration in which the three last indices can be permuted without changing the actual dihedral. However, this only applies to cases where a planar center or tetrahedral center is described. 

The testing code, however, assumed this to be universal. I fixed it by simply requiring that the indices for dihedral angles must be exactly the same in the reference file and the generated file. Perhaps this will cause some problems later but for the current checks it worked fine.